src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,25 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
+
+      </p>
```

Comment:
```suggestion
      <p>
        {' Posted by  '}

        { post.user && <UserInfo user={post.user} />}

      </p>
```
technically post.user can be null and it can make potential errors by reading some fields from null in UserInfo components. suggest to render UserInfo only when post.user exists.